lib/Controller/RoomController.php

```diff
@@ -1443,6 +1443,7 @@ public function joinRoom(string $token, string $password = '', bool $force = tru
 
 	/**
 	 * @PublicPage
+	 * @RequireRoom
```

Comment:
From reading the code, the `getRoomForUserByToken` will fail for the SIP bridge for private rooms because `userId` will be empty.

Reply:
Yeah, that's right but we can remove the user as it only works with qualified SIP requests and that is the authentication already

Reply:
Pushed a fixup